exporters/trace/zipkin/zipkin_test.go

```diff
@@ -389,3 +385,38 @@ func TestErrorOnExportShutdownExporter(t *testing.T) {
 	assert.NoError(t, exp.Shutdown(context.Background()))
 	assert.NoError(t, exp.ExportSpans(context.Background(), nil))
 }
+
+func TestNewExportPipelineWithOptions(t *testing.T) {
+	const eventCountLimit = 10
+
+	collector := startMockZipkinCollector(t)
+	defer collector.Close()
+
+	tp, err := NewExportPipeline(collector.url,
+		WithSDKOptions(
+			sdktrace.WithResource(resource.NewWithAttributes(
+				semconv.ServiceNameKey.String("zipkin-test"),
+			)),
+			sdktrace.WithSpanLimits(sdktrace.SpanLimits{
+				EventCountLimit: eventCountLimit,
+			}),
+		),
+	)
+	require.NoError(t, err)
+
+	otel.SetTracerProvider(tp)
+	_, span := otel.Tracer("zipkin-tracer").Start(context.Background(), "test-span")
+	for i := 0; i < eventCountLimit*2; i++ {
+		span.AddEvent(fmt.Sprintf("event-%d", i))
+	}
+	span.End()
+
+	require.NoError(t, tp.Shutdown(context.Background()))
+
+	checkFunc := func() bool {
+		return collector.ModelsLen() == 1
+	}
+	require.Eventually(t, checkFunc, time.Second, 10*time.Millisecond)
```

Comment:
`tp.Shutdown` should flush all spans, is this needed?

Reply:
Yes, to export span as soon as possible, `tp.Shutdown` is an easy way to do, I think.
However, if this test uses Exporter directly rather than `NewExportPipeline`, it can use `ExportSpans`. If using `tp.Shutdown` is not good, I will change it to use Exporter.

Reply:
I'm confused, TracerProvider.Shutdown will call the SpanProcessor Shutdown which calls the exporter Shutdown.

Have you tried removing this and it failed to receive spans?

Reply:
oops, sorry for my mistake. I confused with `tp.ForceFlush`. I will remove `require.Eventually(t, checkFunc, time.Second, 10*time.Millisecond)`.

Reply:
Done.